### ISSUE TITLE
NTR: Inosine triphosphate high in RBCs (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8345]
New term request Inosine triphosphate high in RBCs (elthat_130531082615): Red blood cells; erythrocytes; ITP

**Comment by pnrobinson** [comment]
@MickeySegal
is this a clinical assay? What is the disease association?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000510
- label: Elevated erythrocyte inosine triphosphate concentration
- parents: [Abnormal erythrocyte metabolite concentration (HP:0034737)](https://hpo.jax.org/app/browse/term/HP:0034737)
- pmid: PMID:12384777
- def: Concentration of inosine triphosphate in red blood cells is above the upper limit of normal.
- comment: Inosine triphosphate pyrophosphohydrolase (ITPase) deficiency is a common inherited condition characterized by the abnormal accumulation of inosine triphosphate (ITP) in erythrocytes.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8345
- synonyms:
